docs/install/lotus/requirements.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Requirements
+description: There are specific system and hardware requirements for running a Lotus node or mining set up.
+---
+
+# Requirements
+
+The requirements of your system will change depending on the type of client you are running: a node, a storage miner, or seal worker.
+
+## Hardware
+
+### Lotus node
+
+Since the Filecoin network is constantly growing and being developed, hardware requirements keep changing. The requirements table below is correct as of writing but may change in the future. The amount of memory (RAM) required is expected to drop significantly as the Lotus client uses Badger FS more efficiently. This is expected to come down to about 1GB in the near future.
+
+| Processor                                      | Memory   | Storage space         |
+| ---------------------------------------------- | -------- | --------------------- |
+| 4 cores<br>8 threads<br> 2.5GHz base frequency | 8GB DDR4 | 200GB (SSD preferred) |
+
+Again, these requirements will change in the near future.
+
+### Storage miner
+
+Miners require a significant amount of hardware to run. The setup below is a minimal example for sealing 32GB sectors on Lotus:
```

Comment:
```suggestion
Miners require a significant amount of hardware to run. The setup below is a minimal example for sealing 32GiB sectors on Lotus:
```

because [sectors are measured in binary](https://docs.filecoin.io/how-to/store/large-files/#maximizing-storage)